### ISSUE TITLE
YTKBatchRequest.h readonly property

### ISSUE THREAD

**Issue by BlankConan** [yuantiku/YTKNetwork#474]
在YTKBatchReqeust.h 有一个属性，是readonly的，如下：
///  All the requests are stored in this array.
@property (nonatomic, strong, readonly) NSArray<YTKRequest *> *requestArray;
但是在.m里面好多地方都有访问或者赋值 _requestArray；
这个是怎么做到的？

**Comment by skyline75489** [comment]
对 requestArray 赋值只有这里吧 https://github.com/yuantiku/YTKNetwork/blob/a2ad5de23ca2b965a90df142fe68627919ccb735/YTKNetwork/YTKBatchRequest.m#L40
